CHANGELOG.md

```diff
@@ -0,0 +1,174 @@
+# CHANGELOG
+
+This is a manually generated log to track changes to the repository for each release. 
+Each section should include general headers such as **Implemented enhancements** 
+and **Merged pull requests**. All closed issued and bug fixes should be 
+represented by the pull requests that fixed them. This log originated with Singularity 2.4
+and changes prior to that are (unfortunately) done retrospectively. From @olifre, critical items to know are:
```

Comment:
Seems sort of weird to shout out olifre here.

Reply:
why? It's his statements. If I don't it's plagarism.

Reply:
Because this is basically documentation and it seems unprofessional.  But if you feel that you would be a plagiarist without the proper citation I won't argue.  I don't feel strongly about it.

Reply:
I removed it! 5 hours ago :)